mos-platform/CMakeLists.txt

```diff
@@ -49,12 +49,12 @@ add_subdirectory(common)
 add_subdirectory(atari2600-common)
 add_subdirectory(atari2600-4k)
 add_subdirectory(atari2600-3e)
-add_subdirectory(atari5200-super)
+add_subdirectory(atari5200-supercart)
```

Comment:
It doesn't seem worthwhile to call out "cart" for targets that have no other way to supply binaries. The benefit that consistency is uniform treatment of indiscernables, but it doesn't seem likely for confusion to occur between targets that support multiple input types and catridge-only machines.

Reply:
Discussed on Discord; this was fine. The canonical name is "Super Cart cartridge", so dropping the improper part of the name does yield "supercart".